dhall/src/Dhall/TypeCheck.hs

```diff
@@ -2269,7 +2281,7 @@ prettyTypeMessage (ListLitInvariant) = ErrorMessages {..}
 
 prettyTypeMessage (InvalidListType expr0) = ErrorMessages {..}
   where
-    short = "Invalid type for ❰List❱"
+    short = "Invalid type for ❰List❱: " <> pretty expr0
```

Comment:
Again I think this is a bit questionable since `expr0` could be large.

Reply:
This can't use diffs.. Perhaps remove this entirely?

Reply:
> Perhaps remove this entirely?

Yeah, I think the type should be visible in the context anyway.

Reply:
Removed in the latest commit.